src/scripts.js

```diff
@@ -82,4 +82,33 @@ function createUserPantry(allIngredientsData) {
 
 function createRecipeRepo(allRecipesData) {
   recipeRepo = new RecipeRepository(allRecipesData);
+}
+
+function addUserIngredients() {
+  console.log(user.id)
+  console.log(user.pantry[0].ingredient)
+  console.log(user.pantry[0].amount)
+  let newData = {"userID": user.id, "ingredientID": user.pantry[0].ingredient, "ingredientModification": user.pantry[0].amount + 5 };
```

Comment:
So glad we got this POST doing something! I'm excited to get better control over its behavior